### ISSUE TITLE
ContextualMenu aria-controls reference and focused element should be a menu

### ISSUE THREAD

**Issue by smhigley** [microsoft/fluentui#17628]
Right now, the `aria-controls` id reference points to a focusable `<div>`, which is also the same element that gets focused when the menu opens.

Both `aria-controls` and the focus target must be the element that has `role="menu"`. There are essentially two options:
1. change the focus target and referenced element to the one that currently has `role="menu"`
2. remove role="menu" from the element that has it, and put it on the <div> that currently gets focused. This would also require adding `role="presentation" to the elements between the new menu and the menuitems.
